src/main/java/Event.java

```diff
@@ -0,0 +1,47 @@
+public class Event extends Task {
+
+    protected String startDay;
+    protected String startTime;
+    protected String endTime;
+
+    /**
+     * Constructor to initialise a task.
+     *
+     * @param description Input based on user.
+     */
+    public Event(String description, String startDay, String startTime, String endTime) {
+        super(description);
+        this.startDay = startDay;
+        this.startTime = startTime;
+        this.endTime = endTime;
```

Comment:
This variable name tells me that this variable is the endTime because it is named endTime!